Marlin/language_tr.h

```diff
@@ -59,11 +62,13 @@
 #define MSG_PREHEAT_1                       _UxGT("Ön Isınma PLA")                                      // Ön Isınma PLA
 #define MSG_PREHEAT_1_N                     MSG_PREHEAT_1 _UxGT(" ")                                    //
 #define MSG_PREHEAT_1_ALL                   MSG_PREHEAT_1 _UxGT(" Tüm")                                 //  Tüm
+#define MSG_PREHEAT_1_END                   MSG_PREHEAT_1 _UxGT(" Nozül")                               //  Nozül
```

Comment:
Uhh, dammit, complete misunderstanding on my side. I derived it from language_en.h and interpreted MSG_PREHEAT_1_**END** as "finished" and not as "hotend".
Sorry for that.